spec/vcr_cassettes/ManageIQ_Providers_CiscoIntersight_PhysicalInfraManager_Refresher/refresh/will_perform_a_full_refresh.yml

```
@@ -33778,6 +33778,185 @@ http_interactions:
           ]
         }
     http_version:
+  recorded_at: Thu, 12 May 2022 13:46:56 GMT- request:
```

This is a YAML syntax error
```suggestion
  recorded_at: Thu, 12 May 2022 13:46:56 GMT
  - request:
```